src/AccessibilityInsights.Extensions.TelemetryTests/OSHelpersUnitTests.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Are we calling test files *UnitTests.cs" or "*Tests.cs" now?

Reply:
The Unit Tests documentation says *UnitTests.cs. The code to run tests looks for *Tests.dll